daemon/libnetwork/drivers/bridge/internal/nftabler/nftabler.go

```diff
@@ -101,7 +101,7 @@ func (nft *nftabler) FilterForwardDrop(ctx context.Context, ipv firewaller.IPVer
 }
```

Comment:
`FilterForwardDrop` receiver isn't updated, so this commit and the next one are currently broken.